doc/source/language/variables.rst

```diff
@@ -604,8 +610,10 @@ duration of the entire file's compile.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Normally the keywords ``local`` and ``global`` can be left off
-as optional in declare statements.  But when you turn LAZYGLOBAL
-off, the compiler starts requiring them to be explicitly stated.
+as optional in declare **identifier** statements.  But when you
```

Comment:
above you have **declare identifier** here you have declare **identifier**, was this intentional?